src/server/pachyderm_test.go

```diff
@@ -689,7 +689,7 @@ func TestPipelineThatOverwritesFile(t *testing.T) {
 	require.Equal(t, 1, len(outCommits))
 	var buffer2 bytes.Buffer
 	require.NoError(t, c.GetFile(outRepo.Name, outCommits[0].Commit.ID, "file", 0, 0, "", nil, &buffer2))
-	require.Equal(t, "foo\nfoo\nfoo\nfoo\nfoo\nfoo\n", buffer2.String())
+	require.Equal(t, "foo\nfoo\nfoo\n", buffer2.String())
```

Comment:
And what is the mechanism to append?

Also, we don't have to worry about API stability yet, but it would be good to start making note of the side effects of the functions in the inline comments for the API docs.